core/tenant/service/secret_service.go

```diff
@@ -71,13 +71,7 @@ func (s SecretService) Update(ctx context.Context, projName tenant.ProjectName,
 	return s.repo.Update(ctx, item)
 }
 
-func (s SecretService) Get(ctx context.Context, projName tenant.ProjectName, namespaceName, name string) (*tenant.PlainTextSecret, error) {
-	secretName, err := tenant.SecretNameFrom(name)
```

Comment:
There is already a check, why do we need to remove it ?

Reply:
since we have already created the secret name in the handler. we can pass it as it is .

also thinking validation should better be done in handlers